<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectDoubleFeedSettingsDialog</class>
 <widget class="QDialog" name="DetectDoubleFeedSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAINSETTING_044</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icon.qrc">
    <normaloff>:/escan2_app.ico</normaloff>:/escan2_app.ico</iconset>
  </property>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TC_PREVIEW_001</string>
   </property>
  </widget>
  <widget class="QLabel" name="Paper_Thickness_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CONFIG_DOC_012</string>
   </property>
  </widget>
  <widget class="PaperThicknessComboBox" name="Paper_Thickness_comboBox">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>19</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Detection_Area_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>51</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CONFIG_DOC_016</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="Det_Area_stackedWidget">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>48</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="mm_page">
    <widget class="QLabel" name="mm_label1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>6</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>EDGEFILLLEVEL_007</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>5</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm_label2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>6</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>EDGEFILLLEVEL_007</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inchi_page">
    <widget class="QLabel" name="inchi_label2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>6</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>EDGEFILLLEVEL_008</string>
     </property>
    </widget>
    <widget class="QLabel" name="inchi_label1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>6</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>EDGEFILLLEVEL_008</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>5</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="DetectionAreaMinDoubleSpinBox" name="Min_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>50</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
  </widget>
  <widget class="DetectionAreaMaxDoubleSpinBox" name="Max_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>50</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Paper_Thickness_colon_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>22</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Detection_Area_colon_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>53</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaperThicknessComboBox</class>
   <extends>QComboBox</extends>
   <header>paperthicknesscombobox.h</header>
  </customwidget>
  <customwidget>
   <class>DetectionAreaMinDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>detectionareamindoublespinbox.h</header>
   <slots>
    <slot>setMaxValue(double)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DetectionAreaMaxDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>detectionareamaxdoublespinbox.h</header>
   <slots>
    <slot>setMinValue(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DetectDoubleFeedSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Paper_Thickness_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>DetectDoubleFeedSettingsDialog</receiver>
   <slot>onValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Max_doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Min_doubleSpinBox</receiver>
   <slot>setMaxValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Min_doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Max_doubleSpinBox</receiver>
   <slot>setMinValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onValueChanged(int)</slot>
  <slot>onValueChanged(double)</slot>
 </slots>
</ui>
